Index: backend/src/main/java/com/steampals/SteamPals/model/UsuarioTieneJuego.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.steampals.SteamPals.model;\r\n\r\npublic class UsuarioTieneJuego {\r\n}\r\n
===================================================================
diff --git a/backend/src/main/java/com/steampals/SteamPals/model/UsuarioTieneJuego.java b/backend/src/main/java/com/steampals/SteamPals/model/UsuarioTieneJuego.java
--- a/backend/src/main/java/com/steampals/SteamPals/model/UsuarioTieneJuego.java	(revision f521062ee2da4349b5f973a3a7ef8c019f8236b8)
+++ b/backend/src/main/java/com/steampals/SteamPals/model/UsuarioTieneJuego.java	(date 1748403965310)
@@ -1,4 +1,17 @@
 package com.steampals.SteamPals.model;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
 public class UsuarioTieneJuego {
+    private float horasJugadas;
 }
